cypress/integration/examples/files.spec.js

```diff
@@ -0,0 +1,115 @@
+/// <reference types="cypress" />
```

Comment:
The file is named 'files.spec.js' but it should match the primary named or default export. Consider renaming the file to match the test description, e.g., 'book_store_app.spec.js'.